src/plasoscaffolder/dal/base_database_information.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+"""Base class for the Information for the SQLite Database"""
+import abc
+
+
+class BaseDatabaseInformation(object):
+  """Base class representing the SQLite Query validator
+  """
+
+  @abc.abstractmethod
+  def getTablesFromDatabase(self) -> [str]:
+    """Executes the SQL Query.
+
+    Returns:
+      [str]: the name of the tables"""
```

Comment:
Does this mean you always want the end """ on a new line? 
I often have arrays/lists as type (in command an type). Do you always want list[x] instead of [x]?